win32/src/file_win32.c

```diff
@@ -0,0 +1,43 @@
+// Copyright (C) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+#include "windows.h"
+#include "file.h"
+#include "file_internal.h"
+
+MU_DEFINE_ENUM_STRINGS(FILE_ASYNC_OPERATION, FILE_ASYNC_OPERATION_VALUES)
+
+typedef struct FILE_HANDLE_DATA_TAG
+{
+    PTP_POOL ptpPool;

```

Comment:
> ptpPool [](start = 12, length = 8)

Can we please be consistent about the notation of the fields?

Either use C style: pool, cleanup_group
Or camel Case: pool, cleanupGroup
Or hungarian notation Windows style: ptpPool, ptpCleanupGroup.

I personally prefer the first, but as long as it is consistent any can be used.

Reply:
Thanks for pointing out. Switching to C-style in next iteration.

---
In reply to: [451984116](https://github.com/Azure/azure-c-pal/pull/7#discussion_r451984116) [](ancestors = 451984116)